src/components/timecard/edit-shift-hours/EditShiftHours.jsx

```diff
@@ -212,12 +219,6 @@ const EditShiftHours = ({
             .setTimeEntryId(responseItem.id)
             .setFinishNextDay(timeEntry.finishNextDay);
 
-          if (
-            startEntryExists &&
-            hasShiftMovedFromTimecard(actualStartDateTime, actualEndDateTime)
-          ) {
-            setMessages(actualStartDateTime, actualEndDateTime);
-          }
           setTimeEntries(newTimeEntries);
           setSummaryErrors({});
```

Comment:
These three lines (`setSummaryErrors({})`, `setShowEditShiftHours(false)` and `setIsErrorVisible(false)`) won't get called if the shift moves from the timecard date, is that something we still want?

Reply:
after this change, the error messages were persisting if the shift moved from timecard, so I moved setSummaryErrors({}) to the top of the onSubmit, so it fixes that - think this works for setIsErrorVisible too perhaps, since they get cleared

and if the shift moves, then I didn't see why the setShowEditShiftHours(false) needed to be called because it disappears from the timecard anyways, but maybe I'm thinking about that wrong?

Reply:
Could we delete `setSummaryErrors({})` from here then if it gets called at the top?
I think you're right for `setShowEditShiftHours(false)`, we probably don't need that

Reply:
yep, that's removed now